modules/Hoot/ui/modals/importDataset.js

```diff
@@ -363,6 +363,16 @@ export default class ImportDataset {
                 Hoot.events.emit( 'modal-closed' );
             } );
     }
+    checkLayerName(layerName) {
```

Comment:
I would move this code to layerManager.js.

I think we also want to use it when doing any new dataset output (conflation & clipping as well as import).
Maybe we could replace the UUID suffix that doesn't give a good sense of which output is most recent.

I think clipping output names just add a number to the end, so maybe we can consolidate the name conflict resolution behaviors so they are all in common (`OutputName_uuid` vs `OutputName_1` vs `OutputName1` could all become `OutputName (1)`)